broker/volumebroker/server/volume_delete.go

```diff
@@ -33,6 +35,19 @@ func (s *Server) DeleteVolume(ctx context.Context, req *ori.DeleteVolumeRequest)
 		return nil, err
 	}
 
+	log.V(1).Info("Deleting encryption secret")
```

Comment:
Please move the log into the following `if` block, otherwise it'd be logged even if nothing happened afterwards.